doc/htmldoc/devices/index.rst

```diff
@@ -1,10 +1,29 @@
 .. _device_index:
 
-All about devices in NEST
-=========================
+All about recording and stimulating networks in NEST
```

Comment:
Please add a link to this document under "More about devices" on get-started_index.html — or alternatively, delete ``devices/index.rst`` page as it contains very little content; just merge the links in to the "More about devices" box on get-started_index.html.